src/python/pants/backend/jvm/tasks/jvm_dependency_usage.py

```diff
@@ -213,7 +214,7 @@ def create_dep_usage_nodes(self, targets, node_creator):
         nodes[concrete_target] = node
 
     # Prune any Nodes with 0 products.
-    for concrete_target, node in nodes.items()[:]:
+    for concrete_target, node in list(nodes.items()):  # copy because mutation
```

Comment:
This was an unusual line to port. The slice was originally there to make a copy of the list. We can achieve the same effect by calling `list()`

I added the comment so people don't accidently remove `list()`